tests_system/lobster_pkg/test_valid_scenario.py

```diff
@@ -0,0 +1,121 @@
+        OUT_FILE = "sample1.lobster"
+        self._test_runner.cmd_args.files = [str(
+            self._data_directory / "sample1.pkg")]
+
+        self._test_runner.cmd_args.out = OUT_FILE
+        completed_process = self._test_runner.run_tool_test()
+
+        # Copy the expected output file to temp directory
+        original_out_file = self._data_directory / OUT_FILE
+        temp_out_file = Path(self.temp_dir.name) / OUT_FILE
+        shutil.copy(original_out_file, temp_out_file)
+        self._replace_working_directory(temp_out_file)
+        self._test_runner.declare_output_file(temp_out_file)
+
+        asserter = Asserter(self, completed_process, self._test_runner)
+
+        asserter.assertNoStdErrText()
+        self.assertIn('Written output for 1 items', completed_process.stdout)
+        asserter.assertExitCode(0)
+        asserter.assertOutputFiles()
+
+    def test_valid_input_pkg_file_without_output_argument(self):
+        OUT_FILE = "report.lobster"
+        self._test_runner.cmd_args.files = [str(
+            self._data_directory / "sample1.pkg")]
+
+        completed_process = self._test_runner.run_tool_test()
+
+        # Copy the expected output file to temp directory
+        original_out_file = self._data_directory / OUT_FILE
+        temp_out_file = Path(self.temp_dir.name) / OUT_FILE
+        shutil.copy(original_out_file, temp_out_file)
+        self._replace_working_directory(temp_out_file)
+        self._test_runner.declare_output_file(temp_out_file)
+
+        asserter = Asserter(self, completed_process, self._test_runner)
+
+        asserter.assertNoStdErrText()
+        self.assertIn('Written output for 1 items', completed_process.stdout)
+        asserter.assertExitCode(0)
+        asserter.assertOutputFiles()
+
+    def test_valid_input_pkg_files(self):
+        OUT_FILE = "sample1_and_sample2.lobster"
+        self._test_runner.cmd_args.files = [str(
+            self._data_directory / "sample1.pkg"),
+            str(self._data_directory / "sample2.pkg")]
+
+        self._test_runner.cmd_args.out = OUT_FILE
+        completed_process = self._test_runner.run_tool_test()
+
+        # Copy the expected output file to temp directory
+        original_out_file = self._data_directory / OUT_FILE
+        temp_out_file = Path(self.temp_dir.name) / OUT_FILE
+        shutil.copy(original_out_file, temp_out_file)
+        self._replace_working_directory(temp_out_file)
+        self._test_runner.declare_output_file(temp_out_file)
+
+        asserter = Asserter(self, completed_process, self._test_runner)
+
+        asserter.assertNoStdErrText()
+        self.assertIn('Written output for 2 items', completed_process.stdout)
+        asserter.assertExitCode(0)
+        asserter.assertOutputFiles()
+
+    def test_valid_input_pkg_folder(self):
+        OUT_FILE = "sample1_folder.lobster"
+        self._test_runner.cmd_args.files = [str(
+            self._data_directory / "pkg_files")]
```

Comment:
Unusual formatting. Please update as follows:
```suggestion
        self._test_runner.cmd_args.files = [
            str(self._data_directory / "pkg_files").
        ]
```